packages/shared/src/components/profile/ProfileForm.tsx

```diff
@@ -123,6 +130,14 @@ export default function ProfileForm({
       const filledFields = Object.keys(data).filter(
         (key) => data[key] !== undefined && data[key] !== null,
       );
+
+      const feedSettings = getLocalFeedSettings(true);
+
+      if (feedSettings) {
+        await updateFeedFilters(feedSettings);
```

Comment:
As mentioned I think we should set the update filter here if the user had any filters.